### PR TITLE
[DCE-154] Publish Artifact

### PR DESCRIPTION
**Pull request by ampats**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Refactor
- [ ] Optimization
- [ ] Test
- [ ] Build
- [x] Documentation Update

## Description
Create an artifact of the web application to download and use on user's local  Windows machine to run the app. 
This artifact should contain 
- [ ] a binary and 
- [ ] a README with instructions to run this application on user's local Windows machine

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Color contrast tested?

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [x] No, and this is why: not needed
- [ ] I need help with writing tests

### CODE REVIEW

**Comment by ampats** [comment]
Should I also fix sln file as shown in last week's call?

File: .github/workflows/practice-basic-webapi-application.yml [hunk 1]
```diff
@@ -13,4 +13,14 @@ jobs:
       
     - name: Build
       run: echo "Hello, World"
```
Comment (sethpthomas91) [inline .github/workflows/practice-basic-webapi-application.yml]:
```suggestion
```
Not needed

File: .github/workflows/practice-basic-webapi-application.yml
Comment (sethpthomas91) [inline .github/workflows/practice-basic-webapi-application.yml:1]: Getting the following warning on your githubaction:
Screenshot 2024-01-29 at 11 23 04 AM
Can we see if we can get rid of that?

File: README-artifact.md
Comment (sethpthomas91) [inline README-artifact.md:1]: Fantastic instructions

**Review by sethpthomas91 — CHANGES_REQUESTED** [review]

**Comment by sethpthomas91** [comment]
Please also fix the name of the PR to be [DCE-154] Publish Artifact

**Comment by sethpthomas91** [comment]
> Should I also fix sln file as shown in last week's call?

You can do this in another PR. Keep your PRs as single focus as possible.

File: README-artifact.md
Comment (ampats) [reply README-artifact.md:1]: Thank you! :)

Comment (sethpthomas91) [inline README-artifact.md:1]: For this file, I would look into some different approaches than keeping two READMEs on the same level. Should there be a release docs directory with this inside? Should these instructions be placed with the overall README and that is just included with the artifact?

**Review by sethpthomas91 — APPROVED** [review]
Great job!

**Pull request merged**